vms/avm/client.go

```diff
@@ -316,12 +314,7 @@ func (c *client) GetTx(ctx context.Context, txID ids.ID, options ...rpc.Option)
 	if err != nil {
 		return nil, err
 	}
-
-	txBytes, err := formatting.Decode(res.Encoding, res.Tx)
-	if err != nil {
-		return nil, err
-	}
-	return txBytes, nil
+	return formatting.Decode(res.Encoding, res.Tx)
```

Comment:
Me: 
Looks at diff -> "this is more than a style nit"...
Looks at code -> "wait... but why would anyone have done this!?"

haha nice catch, 🥇